rocky/reports/templates/partials/export_report_settings.html

```diff
@@ -0,0 +1,127 @@
+            <h2>{% translate "Report name" %}</h2>
+            <p>
+                {% blocktranslate trimmed %}
+                    You can name individual reports.
+                {% endblocktranslate %}
+            </p>
+            <form id="generate_report" class="inline" method="post" action="{{ next }}">
+                <table id="report-name-table">
+                    <caption class="visually-hidden">{% translate "Report names:" %}</caption>
+                    <thead>
+                        <tr>
+                            <th scope="col">{% translate "Name" %}</th>
+                            <th scope="col"></th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for report_name in reports %}
+                            {% with report_id=report_name|slugify %}
+                                <input type="hidden" name="old_report_name" value="{{ report_name }}" />
+                                <tr>
+                                    <td id="text-{{ report_id }}">{{ report_name }}</td>
+                                    <td>
+                                        <input id="{{ report_id }}"
+                                               name="report_name"
+                                               type="hidden"
+                                               readonly
+                                               value="{{ report_name }}" />
+                                    </td>
+                                    <td>
+                                        {% component "modal" size="dialog-medium" %}
+                                        {% fill "trigger" %}
+                                        <button type="button"
+                                                id="edit-name-icon"
+                                                class="icon ti-edit action-button modal-trigger">
+                                            Edit {{ report_name }}
+                                        </button>
+                                    {% endfill %}
+                                    {% fill "header" %}
+                                    <h2>{% translate "Edit " %}{{ report_name }}</h2>
+                                {% endfill %}
+                                {% fill "content" %}
+                                <p>
+                                    {% blocktranslate %}
+                                            Give your report a custom name and optionally add the reports' reference date
+                                            to the name. To do so you can select a standard option or use a
+                                            <a href="https://strftime.org/">Python strftime code</a> in the report name.
+                                        {% endblocktranslate %}
+                                </p>
+                                <form id="content-form" action="" method="">
```

Comment:
It looks like you might have multiple forms with the same ID because of the loop on line 27.
Do we really need a form here at all?

Reply:
Removed form from modal content